apps/token-manager/app/src/web3-utils.js

```diff
@@ -1,10 +1,10 @@
-import { toChecksumAddress } from 'web3-utils'
+import { toChecksumAddress, isAddress } from 'web3-utils'
 
 // Check address equality without checksums
 export function addressesEqual(first, second) {
-  first = first && toChecksumAddress(first)
-  second = second && toChecksumAddress(second)
-  return first === second
+  return isAddress(first) && isAddress(second)
+    ? toChecksumAddress(first) === toChecksumAddress(second)
```

Comment:
We should change the above comment to be just `Check address equality` (since `isAddress()` checks for the checksums), and then just compare them lowercase (checksums already passed with `isAddress()`, and converting to lowercase is faster than to the checksummed version).